projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Boost_Cache.php

```diff
@@ -54,6 +54,38 @@ public function init_actions() {
 		add_action( 'switch_theme', array( $this, 'delete_cache' ) );
 		add_action( 'wp_trash_post', array( $this, 'delete_on_post_trash' ), 10, 2 );
 		add_filter( 'wp_php_error_message', array( $this, 'disable_caching_on_error' ) );
+
+		/**
+		 * Delete all cache.
+		 *
+		 * Allow third-party plugins to clear all cache.
+		 */
+		add_action( 'clear_page_cache_all', array( $this, 'delete_cache' ) );
+
+		/**
+		 * Delete cache for homepage and paged archives.
+		 *
+		 * Allow third-party plugins to clear front-page cache.
+		 */
+		add_action( 'clear_page_cache_front_page', array( $this, 'delete_cache_for_front_page' ) );
```

Comment:
The other actions work as expected, but this action name should be "clear_page_cache_home" to match the others.

```suggestion
		add_action( 'clear_page_cache_home', array( $this, 'delete_cache_for_front_page' ) );
```

Reply:
Updated. But I am curious: what do you mean by matching the others? Are other plugins already using these hooks?

Reply:
The other actions have single word descriptions of what they clear - all, url, post. and home matches what the function uses, get_home().
I'm going to add these actions to WP Super Cache too.